web/client/components/plugins/enhancers/withModulePlugins.js

```diff
@@ -27,7 +27,7 @@ const getPluginsConfig = ({pluginsConfig: config, mode = 'desktop', defaultMode}
  * @param {function(): string[]} getPluginsConfigCallback - callback to extract proper part of plugins configuration passed with `pluginsConfig` prop
  */
 const withModulePlugins = (getPluginsConfigCallback = getPluginsConfig) => (Component) => ({ onLoaded = () => {
-}, pluginsConfig, plugins = {}, loaderComponent = () => null, ...props }) => {
+}, pluginsConfig, plugins = {}, loaderComponent = false, ...props }) => {
```

Comment:
loaderComponent must be a component or undefined.

```suggestion
}, pluginsConfig, plugins = {}, loaderComponent, ...props }) => {
```